Dockerfile.fastapi

```
@@ -0,0 +1,115 @@
+# ============================================================================
+FROM python:3.12-slim
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONIOENCODING=utf-8 \
+    PATH="/opt/venv/bin:$PATH"
+
+# Install only runtime dependencies (curl for healthcheck, ca-certificates for HTTPS)
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)